mods/measure/init.lua

```diff
@@ -0,0 +1,50 @@
+local function deltas_as_string(pos1, pos2)
+	return("Deltas: X: "..math.abs(pos1.x-pos2.x).." | Y: "..math.abs(pos1.y-pos2.y).." | Z: "..math.abs(pos1.z-pos2.z))
+	end
+
+minetest.register_craftitem("measure:stick", {
+
+	description = "Measuring Stick",
+	inventory_image = "measure_stick.png",
+	stack_max = 1,
+
+	on_use = function(itemstack, user, pointed_thing)
+
```

Comment:
Add the following at the beginning of `on_use` to ensure `user` actually exists, as `user` can also be `nil` as explained [here](https://github.com/minetest/minetest/blob/9854340c0b21d01a94ba7ef526c775e59e37b898/doc/lua_api.txt#L5815)

```lua
if not user then
	return
end
```

Reply:
Make sure to add this before `local name = user:get_player_name()` as this could otherwise result in a `nil` access if `user` is `nil`